### PR TITLE
COMMERCE-829 Add data sources for categories and tags

### PR DESCRIPTION
**Pull request by ethanbustad**
https://issues.liferay.com/browse/COMMERCE-829

### CODE REVIEW

**Comment by ethanbustad** [comment]
ci:test

**Comment by ethanbustad** [comment]
ci:test

File: commerce-product-service/src/main/java/com/liferay/commerce/product/internal/data/source/CPDataSourceAssetCategoriesAndTagsImpl.java [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+/**
+ * @author Marco Leo
+ */
+@Component(
+	immediate = true,
+	property = "commerce.product.data.source.name=" + CPDataSourceAssetCategoriesAndTagsImpl.NAME,
+	service = CPDataSource.class
+)
+public class CPDataSourceAssetCategoriesAndTagsImpl implements CPDataSource {
```
Comment (marco-leo) [inline commerce-product-service/src/main/java/com/liferay/commerce/product/internal/data/source/CPDataSourceAssetCategoriesAndTagsImpl.java]: @ethanbustad Can we use a base class with the common logic?

Comment (ethanbustad) [reply commerce-product-service/src/main/java/com/liferay/commerce/product/internal/data/source/CPDataSourceAssetCategoriesAndTagsImpl.java]: Sure!

**Pull request merged**